BLD: wheel building failing

https://travis-ci.org/MacPython/pandas-wheels/jobs/504201466

just started happening. I guess we are not running 1.12 on the 2.7 build, so need to skip this test.

```
================================== FAILURES ===================================
_______ TestDataFrameConstructors.test_constructor_maskedrecarray_dtype ________
[gw1] linux2 -- Python 2.7.6 /venv/bin/python
self = <pandas.tests.frame.test_constructors.TestDataFrameConstructors object at 0xd9592c2c>
    def test_constructor_maskedrecarray_dtype(self):
        # Ensure constructor honors dtype
        data = np.ma.array(
            np.ma.zeros(5, dtype=[('date', '<f8'), ('price', '<f8')]),
            mask=[False] * 5)
>       data = data.view(ma.mrecords.mrecarray)
E       AttributeError: 'module' object has no attribute 'mrecords'
/venv/local/lib/python2.7/site-packages/pandas/tests/frame/test_constructors.py:797: AttributeError
_____________ TestDataFrameConstructors.test_constructor_mrecarray _____________
[gw1] linux2 -- Python 2.7.6 /venv/bin/python
self = <pandas.tests.frame.test_constructors.TestDataFrameConstructors object at 0xd95ba3cc>
    def test_constructor_mrecarray(self):
        # Ensure mrecarray produces frame identical to dict of masked arrays
        # from GH3479
    
        assert_fr_equal = functools.partial(tm.assert_frame_equal,
                                            check_index_type=True,
                                            check_column_type=True,
                                            check_frame_type=True)
        arrays = [
            ('float', np.array([1.5, 2.0])),
            ('int', np.array([1, 2])),
            ('str', np.array(['abc', 'def'])),
        ]
        for name, arr in arrays[:]:
            arrays.append(('masked1_' + name,
                           np.ma.masked_array(arr, mask=[False, True])))
        arrays.append(('masked_all', np.ma.masked_all((2,))))
        arrays.append(('masked_none',
                       np.ma.masked_array([1.0, 2.5], mask=False)))
    
        # call assert_frame_equal for all selections of 3 arrays
        for comb in itertools.combinations(arrays, 3):
            names, data = zip(*comb)
>           mrecs = ma.mrecords.fromarrays(data, names=names)
E           AttributeError: 'module' object has no attribute 'mrecords'
/venv/local/lib/python2.7/site-packages/pandas/tests/frame/test_constructors.py:826: AttributeError
=============================== warnings summary ===============================
```

--- a/pandas/tests/frame/test_constructors.py
+++ b/pandas/tests/frame/test_constructors.py
@@ -21,7 +21,7 @@ from pandas.core.dtypes.common import is_integer_dtype
 import pandas as pd
 from pandas import (
     Categorical, DataFrame, Index, MultiIndex, Series, Timedelta, Timestamp,
-    compat, date_range, isna)
+    compat, date_range, isna, _np_version_under1p13)
 from pandas.tests.frame.common import TestData
 import pandas.util.testing as tm
 
@@ -684,6 +684,8 @@ class TestDataFrameConstructors(TestData):
         frame = DataFrame(['foo', 'bar'], index=[0, 1], columns=['A'])
         assert len(frame) == 2
 
+    @pytest.mark.skipif(PY2 & _np_version_under1p13,
+                        reason="old numpy & py2")
     def test_constructor_maskedarray(self):
         self._check_basic_constructor(ma.masked_all)
 

--- a/pandas/tests/frame/test_constructors.py
+++ b/pandas/tests/frame/test_constructors.py
@@ -21,7 +21,7 @@ from pandas.core.dtypes.common import is_integer_dtype
 import pandas as pd
 from pandas import (
     Categorical, DataFrame, Index, MultiIndex, Series, Timedelta, Timestamp,
-    compat, date_range, isna, _np_version_under1p13)
+    _np_version_under1p13, compat, date_range, isna)
 from pandas.tests.frame.common import TestData
 import pandas.util.testing as tm
 

